account_loan/models/account_loan.py

```diff
@@ -396,7 +397,7 @@ def _compute_draft_lines(self):
                 self._new_line_vals(i, date, amount)
             )
             line._check_amount()
-            date += delta
+            date += delta + relativedelta(day=initial_date.day)
```

Comment:
This solution is not correct at all. I think it's better:

```suggestion
            date = initial_date + delta * i
```

Reply:
@etobella do you prefer my option?